pyogrio/tests/test_geopandas_io.py

```diff
@@ -2067,3 +2067,22 @@ def test_non_utf8_encoding_shapefile_sql(tmp_path, use_arrow):
     )
     assert actual.columns[0] == mandarin
     assert actual[mandarin].values[0] == mandarin
+
+
+def test_write_kml_file_coordinates(tmp_path):
+    # confirm reading with kml stores as lon/lat convention
+    gdf = gp.GeoDataFrame(
```

Comment:
```suggestion
    points = [Point(10, 20), Point(30, 40), Point(50, 60)]
    gdf = gp.GeoDataFrame(geometry=points, crs="EPSG:4326")
```
(no need for attributes or `set_crs()` later)